test/python/WMCore_t/MicroService_t/MSRuleCleaner_t/MSRuleCleaner_t.py

```diff
@@ -392,9 +393,23 @@ def testPipelineArchiveStepChain(self):
         with self.assertRaises(MSRuleCleanerArchivalSkip):
             self.msRuleCleaner.plineArchive.run(wflow)
 
-    def testRunning(self):
-        result = self.msRuleCleaner._execute(self.reqRecords)
-        self.assertEqual(result, (3, 2, 0, 0))
+    def testMemoryLeak(self):
```

Comment:
You probably didn't want to persist this memory leak investigation as a unit test (?)